WBooks/BarControllers/NavBarController.swift

```diff
@@ -11,9 +11,6 @@ import WolmoCore
 
 class NavBarController: UINavigationController {
     override func viewDidLoad() {
-        
-        super.viewDidLoad()
```

Comment:
You shouldn't delete this.  All the functions that are about viewController's life cycle should have the `super` function call